### DISCUSSION TITLE
two slider HTML-response plugin (with option to show slider number as slider moves)

### DISCUSSION THREAD

**Discussion by samanthareisman** [jspsych/jsPsych#2940] [Show and tell]
Hi all, 

I wanted to share this plugin I made: [https://github.com/samanthareisman/jsPsych-plugins/blob/main/plugin-html-2slider-response.js](url)

It allows you to include two sliders for an HTML stimulus (with separate options for `min`, `max`, `labels`, `steps`, `width`, `slider_start`, `prompt`). In addition, there is an added parameter for each slider called `slider_number` & `slider_number2` that when set to `true` (`default: false`) will display the slider number and change as the slider moves around. 

(You can also find a similar plugin but for a canvas stimulus here: [https://github.com/samanthareisman/jsPsych-plugins](url))

Special thanks to @gorkang and Discussion #1147 for the code that allows for the dynamic slider number display!

**Comment by jodeleeuw** [comment]
This is great @samanthareisman!

Feel free to contribute this to [jspsych-contrib](https://github.com/jspsych/jspsych-contrib)!

**Reply by aaclavijoa** [reply]
The link above does not take the user, in this case I am the user, to the code.
